exceptions_hog/handler.py

```diff
@@ -196,6 +197,16 @@ def exception_handler(
     ):
         # By default don't handle non-DRF errors in DEBUG mode, i.e. Django will treat
         # unhandled exceptions regularly (very evident yellow error page)
+
+        # NOTE: to make sure we get exception tracebacks in test responses, we need
+        # to make sure this signal is called. The django test client uses this to
+        # pull out the exception traceback.
+        #
+        # See https://github.com/django/django/blob/master/django/test/client.py#L714
```

Comment:
Since a file on the master branch can change a lot over time, I would suggest changing the link to point to a specific version of django and thus prevent the link from losing its original meaning.


```suggestion
        # See https://github.com/django/django/blob/3.2.9/django/test/client.py#L712
```

Reply:
Thanks @luzfcb, great suggestion!